lib/phonebook/ui/pages/association_editor_page/member_editable_card.dart

```diff
@@ -0,0 +1,148 @@
+import 'package:myecl/phonebook/ui/pages/admin_page/edition_button.dart';
+import 'package:myecl/tools/functions.dart';
+import 'package:myecl/user/providers/profile_picture_provider.dart';
+import 'package:myecl/phonebook/tools/constants.dart';
+import 'package:qlevar_router/qlevar_router.dart';
+
+class MemberEditableCard extends HookConsumerWidget {
+  const MemberEditableCard({super.key, required this.member});
+
+  final CompleteMember member;
+
+  @override
+  Widget build(BuildContext context, WidgetRef ref) {
+    final profilePicture = ref.watch(profilePictureProvider);
+    final association = ref.watch(associationProvider);
+    final associationNotifier = ref.watch(asyncAssociationProvider.notifier);
+    final associationMembersNotifier =
+        ref.watch(associationMemberListProvider.notifier);
+    final roleTagsNotifier = ref.watch(rolesTagsProvider.notifier);
+    final editionNotifier = ref.watch(editionProvider.notifier);
+    final completeMemberNotifier = ref.watch(completeMemberProvider.notifier);
+    void displayToastWithContext(TypeMsg type, String msg) {
+      displayToast(context, type, msg);
+    }
+
+    return Container(
+        padding: const EdgeInsets.all(10),
+        margin: const EdgeInsets.symmetric(horizontal: 30, vertical: 10),
+        decoration: BoxDecoration(
+          border: Border.all(),
+          color: Colors.white,
+          borderRadius: const BorderRadius.all(Radius.circular(20)),
+        ),
+        child: Row(
+          children: [
+            Container(
+              decoration: BoxDecoration(
+                shape: BoxShape.circle,
+                boxShadow: [
+                  BoxShadow(
+                    color: Colors.black.withOpacity(0.1),
+                    spreadRadius: 5,
+                    blurRadius: 10,
+                    offset: const Offset(2, 3),
+                  ),
+                ],
+              ),
+              child: profilePicture.when(
+                data: (picture) {
+                  return CircleAvatar(
+                    radius: 20,
+                    backgroundImage: picture.isEmpty
+                        ? const AssetImage('assets/images/logo_alpha.png')
+                        : Image.memory(picture).image,
+                  );
+                },
+                loading: () {
+                  return const Center(
+                    child: CircularProgressIndicator(),
+                  );
+                },
+                error: (e, s) {
+                  return const Center(
+                    child: Text("Error"),
+                  );
+                },
+              ),
+            ),
+            const SizedBox(width: 10),
+            SizedBox(
+              width: 400,
+              child: Column(
+                children: [
+                  Text(
+                    "${member.member.name} ${member.member.firstname} (${member.member.nickname})",
```

Comment:
What if the member nickname is empty ? Consider using the getNamed herited function from SimpleUser